### PR TITLE
Fix failing Dockerfile build

### PR DESCRIPTION
**Pull request by dehidehidehi**
The `Dockerfile` builds properly.
However, and this is probably a seperate issue : I'm having trouble making any `task sync` command work properly.

### CODE REVIEW

**Comment by dehidehidehi** [comment]
The `Dockerfile` builds properly.
However, and this is probably a seperate issue : I'm having trouble making any `task sync` command work properly.

**Comment by dehidehidehi** [comment]
See #102

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,9 @@
...
 COPY . .
-RUN apt-get update && apt-get -y install sudo && apt-get install -y git
+RUN apt-get update && apt-get -y install apt-utils && apt-get -y install sudo && apt-get install -y git
 RUN ln -snf /usr/share/zoneinfo/$CONTAINER_TIMEZONE /etc/localtime && echo $CONTAINER_TIMEZONE > /etc/timezone
-RUN chmod +x tw-install.sh && ./tw-install.sh
\ No newline at end of file
+RUN chmod +x tw-install.sh && ./tw-install.sh
+ARG TASKDDATA
+ENV TASKDDATA ${TASKDDATA:-/var/taskd}
+ENTRYPOINT taskd server --data ${TASKDDATA}
```
Comment (dehidehidehi) [inline Dockerfile:9]: The docker image entrypoint starts the `taskd` server.
Which was not the case previously.

**Comment by a-t-0** [comment]
Thank you for your effort towards restoring the dockerfile functionality. I will merge it as I believe it improves the current situation. For completeness, when I run:
`docker build .`
or
`docker build -t tw-install .`
I get the following errors:
```sh
ouch: cannot touch 'src/taskd.service': No such file or directory
sed: can't read src/taskd.service: No such file or directory
src/D_configure_taskserver.sh: line 177: src/taskd.service: No such file or directory
touch: cannot touch 'src/taskd.service': No such file or directory
sed: can't read src/taskd.service: No such file or directory
src/D_configure_taskserver.sh: line 177: src/taskd.service: No such file or directory
touch: cannot touch 'src/taskd.service': No such file or directory
sed: can't read src/taskd.service: No such file or directory
src/D_configure_taskserver.sh: line 177: src/taskd.service: No such file or directory
touch: cannot touch 'src/taskd.service': No such file or directory
sed: can't read src/taskd.service: No such file or directory

src/D_configure_taskserver.sh: line 177: src/taskd.service: No such file or directory
cp: cannot stat 'src/taskd.service': No such file or directory
output=
sudo: systemctl: command not found
output=
sudo: systemctl: command not found
output=
src/D_configure_taskserver.sh: line 219: systemctl: command not found
output=
sudo: systemctl: command not found
output=
40474A55477F0000:error:80000002:system library:BIO_new_file:No such file or directory:../crypto/bio/bss_file.c:67:calling fopen(/home/root/.task/ca.cert.pem, r)
40474A55477F0000:error:10000080:BIO routines:BIO_new_file:no such file:../crypto/bio/bss_file.c:75:
40474A55477F0000:error:05880002:x509 certificate routines:X509_load_cert_crl_file_ex:system lib:../crypto/x509/by_file.c:228:
Cannot open /home/root/.task/ca.cert.pem for reading
Cannot open /home/root/.task/first.cert.pem for reading
Cannot open /home/root/.task/ca.cert.pem for reading
Cannot open /home/root/.task/first.cert.pem for reading
Note that no verification profile was selected. In the future the medium profile will be enabled by default.
Use --verify-profile low to apply the default verification of NORMAL priority string.
Loaded CAs (1 available)
	Subject: CN=dda5cd1ae81c,O=Göteborg Bit Factory
	Issuer: CN=dda5cd1ae81c CA,O=Göteborg Bit Factory,L=Göteborg,ST=Västra Götaland,C=SE
	Checked against: CN=dda5cd1ae81c CA,O=Göteborg Bit Factory,L=Göteborg,ST=Västra Götaland,C=SE
	Signature algorithm: RSA-SHA256
	Output: Verified. The certificate is trusted. 

Chain verification output: Verified. The certificate is trusted. 

Note that no verification profile was selected. In the future the medium profile will be enabled by default.
Use --verify-profile low to apply the default verification of NORMAL priority string.
Loaded CAs (1 available)
	Subject: CN=dda5cd1ae81c,O=Göteborg Bit Factory
	Issuer: CN=dda5cd1ae81c CA,O=Göteborg Bit Factory,L=Göteborg,ST=Västra Götaland,C=SE
	Checked against: CN=dda5cd1ae81c CA,O=Göteborg Bit Factory,L=Göteborg,ST=Västra Götaland,C=SE
	Signature algorithm: RSA-SHA256
	Output: Verified. The certificate is trusted. 

Chain verification output: Verified. The certificate is trusted. 

	Subject: CN=dda5cd1ae81c,O=Göteborg Bit Factory
subject=O = G\C3\B6teborg Bit Factory, CN = dda5cd1ae81c
src/run_taskserver_at_boot.sh: line 4: crontab: command not found
src/run_taskserver_at_boot.sh: line 11: crontab: command not found
Cron job successfully set up.
src/run_taskserver_at_boot.sh: line 14: crontab: command not found
Error, cronjob was not found.

```
and when I run:
```bash
docker run \
-p 53589:53589 \
-e TW_USERNAME=First \
-e TW_ORGANISATION=Public \
--name tw-instance \
-d \
tw-install
```
I get:
```
Unable to find image 'tw-install:latest' locally
docker: Error response from daemon: pull access denied for tw-install, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.
```
I appreciate you took the time to install cron inside the docker, and once again thank you for your effort, I greatly appreciate it!

**Pull request merged**